cli/app.php

```diff
@@ -641,19 +641,21 @@ function (ConsoleCommandEvent $event) {
     /**
      * Proxy commands through to an isolated site's version of PHP.
      */
-    $app->command('php [command]', function (OutputInterface $output, $command) {
+    $app->command('php [--site=] [command]', function (OutputInterface $output, $command) {
```

Comment:
```suggestion
    $app->command('php [command] [--site=]', function (OutputInterface $output, $command) {
```

Reply:
ah okay. Yeah that's fine. Bit odd though : P

Reply:
Yah.. I'm still wondering whether I should just do the work to make it possible to pass the `--site=` anywhere. It's totally doable, it's just.. gonna be really ugly in Bash.